extensions/configuration-editing/package.json

```diff
@@ -140,11 +140,11 @@
       },
       {
         "fileMatch": "devcontainer.json",
-        "url": "./schemas/devContainer.schema.generated.json"
+        "url": "./schemas/devContainer.schema.json"
```

Comment:
Couldn't we reference the original like https://github.com/microsoft/vscode/blob/main/extensions/html-language-features/package.json#L253 does?

Reply:
IMO it's better to avoid fetching the schema over the web so that all works offline